x-pack/plugins/apm/server/routes/mobile/get_mobile_main_statistics_by_field.ts

```diff
@@ -55,7 +63,7 @@ export async function getMobileMainStatisticsByField({
 }: Props) {
   async function getMobileTransactionEventStatistics() {
```

Comment:
```
 getMobileMainStatisticsByField(){
    async function getMobileTransactionEventStatistics() {}
    async function getMobileErrorEventStatistics(){}
}
```

Nesting functions can be useful for encapsulating functionality and scoping variables but in this case, it seems it doesn't provide any value. 

On the other hand, if there is a possibility of reusing `ex getMobileErrorEventStatistics` in other parts of the codebase, defining it outside of `getMobileMainStatisticsByField` could be a more practical approach. 

What do you think about defining them outside of `getMobileMainStatisticsByField`?

Reply:
I thought about it, but `getMobileErrorEventStatistics` is getting crash errors **by field**, it's not generic enough to be reusable. But I will keep it in mind for when we start implementing more use cases for `crashRate`

Reply:
> but getMobileErrorEventStatistics is getting crash errors by field, it's not generic enough to be reusable.

tbh, I'm not sure what you meant with it  🤔 but I think something like the following _will work_  

```
const commonPros = {
    kuery,
    apmEventClient,
    serviceName,
    environment,
    start,
    end,
    field,
  };
  const [transactioEventStatistics, errorEventStatistics] = await Promise.all([
    getMobileTransactionEventStatistics(commonPros),
    getMobileErrorEventStatistics(commonPros),
  ]);
```